diff --git a/modified_scripts/multi_disk_fio_scripts-V0.4/ssd_raw_fio_test.sh b/modified_scripts/multi_disk_fio_scripts-V0.4/ssd_raw_fio_test.sh
index fc1b8e5..e3e1699 100644
--- a/modified_scripts/multi_disk_fio_scripts-V0.4/ssd_raw_fio_test.sh
+++ b/modified_scripts/multi_disk_fio_scripts-V0.4/ssd_raw_fio_test.sh
@@ -12,8 +12,8 @@ fi
 devname="$1"
 #Sequence write/read test of ssd
 "$SCRIPT_DIR/raw_disk_fio_test.sh" -b -d "$devname"
-sleep 30s
+sleep 30
 
 #Random write/read test of ssd
 "$SCRIPT_DIR/raw_disk_fio_test.sh" -B -d "$devname"
-sleep 30s
+sleep 30
