diff --git a/modified_scripts/multi_disk_fio_scripts-V0.4/hdd_raw_fio_test.sh b/modified_scripts/multi_disk_fio_scripts-V0.4/hdd_raw_fio_test.sh
index 64f255c..58c3629 100644
--- a/modified_scripts/multi_disk_fio_scripts-V0.4/hdd_raw_fio_test.sh
+++ b/modified_scripts/multi_disk_fio_scripts-V0.4/hdd_raw_fio_test.sh
@@ -12,8 +12,8 @@ fi
 devname="$1"
 #Sequence write/read test of HDD
 "$SCRIPT_DIR/raw_disk_fio_test.sh" -v -d "$devname"
-sleep 30s
+sleep 30
 
 #Random write/read test of HDD
 "$SCRIPT_DIR/raw_disk_fio_test.sh" -V -d "$devname"
-sleep 30s
+sleep 30
